FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MedianComputable.java

```diff
@@ -13,6 +17,20 @@ public float compute(float[] values) {
         return (values[(n)/2-1] + values[n /2]) / 2;
     }
 
+    private void getInputUnits() {
+        if(config == null || config.getUnits().isEmpty()) {
+            outputUnit = DEFAULT_INPUT_UNITS;
```

Comment:
This is weird to me. Why does getInputUnits set output unit and not return anything? potential copy paste error going on her?